experimental/zerotrie/src/zerotrie.rs

```diff
@@ -61,6 +61,8 @@ use litemap::LiteMap;
 /// # Ok::<_, zerotrie::ZeroTrieError>(())
 /// ```
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
+// Note: The absence of the following derive does not cause any test failures in this crate
```

Comment:
I don't think we should add the yokeable implementation, data structs need to be yokeable, but their fields don't. We never want to use a raw collection as a data struct, and this crate should be independent of our mechanism of lifetime erasure.

Reply:
Nevermind, field also need to be yokeable (I'm not sure they have to though?)